cypress/integration/dictionary/addConceptInDictionaryPage.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Add concepts to an existing dictioanry
+  Background:
+    Given the user is logged in
```

Comment:
I couldn't find the step implementation for this one.

Reply:
We have a small number of steps that can be used anywhere because they would have basically the same definition everywhere. In this case, the implementation is [here](https://github.com/openmrs/openmrs-ocl-client/blob/72e2dfe39c12a5d6f3ec4dd131d5611a14713a41/cypress/support/step_definitions/common/common.ts#L9).